src/content/turbosnap/setup-turbosnap.mdx

```diff
@@ -68,6 +78,43 @@ You may need additional config in the following situations:
 
 Jump to the [recipes](#recipes) section for more information.
 
+### Update your configuration using TurboSnap Helper
+
+Having TurboSnap properly configured is essential when trying to optimize build performance. If you're working in a monorepo, or even just a large project with a complex setup, setting up TurboSnap efficiently can feel overwhelming. That's where the TurboSnap helper comes in! We've introduced the `@chromatic-com/turbosnap-helper` CLI utility to simplify the process of managing your configuration.
+
+The TurboSnap Helper utility is designed to assist with properly configuring TurboSnap for Chromatic. It's especially helpful in monorepos, where managing directories, static assets, and build optimizations across multiple projects can get tricky.
+
+Run the utility from the root of your repo using:
+
+```sh
+npx @chromatic-com/turbosnap-helper
+```
+
+<div class="aside">
+
+❓ **Why run the utility from the repo root rather than the project root?**
+TurboSnap Helper has been updated to support monorepos! By starting from the root, it can scan across all of your projects, so you're not accidentally missing assets or settings that could cause full rebuilds. If you run the utility from your project root, not a problem! The utility will still work just fine.
```

Comment:
```suggestion
TurboSnap Helper supports monorepos. By starting from the root, it can scan across all of your projects, so you're not accidentally missing assets or settings that could cause full rebuilds. If you run the utility from your project root, not a problem. The utility will still work just fine.
```